### PR TITLE
Assign screen

### PR DESCRIPTION
**Pull request by Mary-Harshitha**
image
image
image
image

### CODE REVIEW

File: src/components/Malta/RecommendedAssignment.tsx [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+  const handleChange2 = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setChecked([event.target.checked, event.target.checked]);
+  };
+
+  return (
+    //render .map()
+    <div className="display-card">
+      <div style={{ fontSize: "8px" }}>
+        These are the recommended assignments based on the previous assignments
```
Comment (bhanugummana) [inline src/components/Malta/RecommendedAssignment.tsx:30]: Give i18n support for the text

File: src/components/Malta/RecommendedAssignment.tsx [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+            <Checkbox
+              checked={checked[0]}
+              onChange={handleChange1}
+              size="small"
+            />
+          }
+        />
+      </div>
+      <div className="recommended-content">
+        <div className="recommended-subject-header">
```
Comment (bhanugummana) [inline src/components/Malta/RecommendedAssignment.tsx:46]: create multiple components

File: src/pages/Malta/Assignment.tsx [hunk 3]
```diff
@@ -0,0 +1,337 @@
...
+import RecommendedAssignment from "../../components/Malta/RecommendedAssignment";
+import AssignButton from "../../components/Malta/AssignButton";
+
+interface TabPanelProps {
+  children?: React.ReactNode;
+  index: number;
+  value: number;
+}
+
+function CustomTabPanel(props: TabPanelProps) {
```
Comment (bhanugummana) [inline src/pages/Malta/Assignment.tsx:43]: create components files

**Review by bhanugummana — CHANGES_REQUESTED** [review]
Add  screenshot of the sample UI

File: src/components/Malta/assignment/AssignButton.tsx [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+interface AssignButtonProps {
+  onClicked: React.MouseEventHandler<HTMLIonButtonElement>;
+  disabled: boolean;
+}
+
+const AssignButton: React.FC<AssignButtonProps> = ({ onClicked, disabled }) => {
+  return (
+    <div
+      style={{
+        display: "flex",
```
Comment (bhanugummana) [inline src/components/Malta/assignment/AssignButton.tsx:16]: Add style in css  files

File: src/components/Malta/assignment/DisplayLesson.tsx [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+                  return l;
+                });
+
+                setAllLessons(newLessons);
+                onLessonSelect(lesson);
+              }}
+              className="lesson-button"
+              key={lesson.lesson.id}
+            >
+              <div className="lesson-icon-and-chapter-download-container">
```
Comment (bhanugummana) [inline src/components/Malta/assignment/DisplayLesson.tsx:57]: Divide them into multiple components

File: src/components/Malta/assignment/RecommendedAssignment.tsx [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+                          <IonCardSubtitle>{t(chapter.title)}</IonCardSubtitle>
+                        </div>
+                      </IonCardHeader>
+                      <IonCardContent>
+                        {chapter.lessons.map((lesson) => {
+                          return (
+                            <div
+                              style={{
+                                display: "flex",
+                                flexDirection: "column",
```
Comment (bhanugummana) [inline src/components/Malta/assignment/RecommendedAssignment.tsx:90]: Add style in css file

File: src/components/Malta/assignment/RecommendedAssignment.tsx [hunk 7]
```diff
@@ -0,0 +1,115 @@
...
+} from "@ionic/react";
+
+const RecommendedAssignment: React.FC<{
+  infoText: string;
+}> = ({ infoText }) => {
+  const subjects = [
+    {
+      chapters: [
+        {
+          lessons: [
```
Comment (bhanugummana) [inline src/components/Malta/assignment/RecommendedAssignment.tsx:20]: Is this dummy data??

**Review by bhanugummana — CHANGES_REQUESTED** [review]
Rename the folder Malta -> malta

File: src/components/Malta/assignment/RecommendedAssignment.tsx (hunk 7 above)
Comment (Mary-Harshitha) [reply src/components/Malta/assignment/RecommendedAssignment.tsx:20]: yes

**Pull request merged**